### ISSUE TITLE
New Behavior - Crashes After Loading in Browser

### ISSUE THREAD

**Issue by tentious** [DrLex0/FlashAirUI#3]
I originally set up FlashAirUI well over 2-3 years ago and use this for transferring files to the 3D Printer. All has worked well for years and suddenly when I attempt to load the UI, the SD Card wifi seems to crash. Nothing major has changed in my environment(i.e. no new WAP).

I updated to latest release, same behaviour.

Steps I've gone through so far:

I plug in the SD Card - I'm able to ping the device indefinately.
As soon as trying to access the UI via Chrome or Internet Explorer(Not Chromium), pings stop responding and chrome/IE only loads the title in the title bar of the browser window. In development mode, checking network tab, I can see that jquery.js, main.js, and spin.gif loads.

I've attached a screenshot.

![image](https://user-images.githubusercontent.com/1422016/114284333-c9c3ae00-9a14-11eb-84c8-7c71354962aa.png)

**Comment by DrLex0** [comment]
So you have changed nothing about the card, and it suddenly started acting up? It is very unlikely that this has anything to do with the FlashAirUI because it only consists of a HTML page and some scripts. It could be the card itself or the power source that developed a problem.
Have you tried accessing the UI when the card is mounted in another SD card slot than the one of your printer?

**Comment by tentious** [comment]
Sorry for the long delay. Nothing was changed. In fact, I even tried formatting/reload and still have the problem. I believe it may be the card at this point. Again, it's served me well over the past few years and time for a replacement. Very strange as it seems to partially load and then crashes completely. It is however still assessible via SD card reader. Any other ideas other than tossing it for another?

**Comment by DrLex0** [comment]
I'm afraid the card is toast, unless it's the SD card slot in your printer that has problems providing enough power for the WiFi functionality. You can try putting the card in another device and then trying to access the web UI. If it also crashes there, then it's the card. If not, then it's the slot of the printer that has a problem (which would be more annoying than a problem with the card of course).

**Comment by tentious** [comment]
Also tried in two different readers and same result. What's more interesting is that if I create a list.htm file with just some basic text, the text loads in the browser and card responds to pings. Seems to be related to serving the javascript code. As you suggest, likely the card as I changed nothing and it started this behavior.

As some last resorts, I also tried updating to latest firmware(thinking maybe backend OS was corrupt or something), but get the same result - crashes immediately after I attempt to load the FlashAirUI.

I'll move on to Octoprint or some other utility.

Thanks again for such an awesome UI to the card that's served me well since 2017ish.
